packages/http-client-csharp/generator/TestProjects/CadlRanch/http/type/property/nullable/src/Generated/Models/BytesProperty.cs

```diff
@@ -8,8 +8,18 @@ namespace _Type.Property.Nullable.Models
 {
     public partial class BytesProperty
     {
-        public string RequiredProperty => throw null;
+        public BytesProperty(string requiredProperty, BinaryData nullableProperty) => throw null;
```

Comment:
do these changes come from some differences from the spec?

Reply:
I think it from TCGC change maybe. I confirmed this is roundtrip model. It's just the input usage is in a patch operation. Maybe previously we don't take patch input as input, but now we take it as input.

Reply:
if your PR is updating TCGC version, this might be the reason.
But I did not see you change TCGC version, why this change could possibly come from TCGC change?
Could you compare the spec from cadl-ranch and the new position to see if there is any differences? If the spec is the same, we need a reasonable reason for this change.

Reply:
You are correct. This file is missing in the new spec https://github.com/Azure/cadl-ranch/blob/main/packages/cadl-ranch-specs/http/type/property/nullable/client.tsp. I'm confirming if it is expected.

Reply:
as I said above, if you cannot get a confirmation and want to merge this PR, please create an issue to track it (with context)

Reply:
https://github.com/microsoft/typespec/issues/5399